src/ui/Roster.ts

```diff
@@ -79,6 +84,78 @@ export default class Roster {
          spanElement.text(msg);
       }
    }
+   
+   private initCollapisible()
+   {
+      let coll = $(".jsxc-collapsible");
+      let i;
+
+      for (i = 0; i < coll.length; i++)
+      {
+         $(coll[i]).off("click").on("click", function()
+         {
+            this.classList.toggle("jsxc-active");
+            let content = $(this).next();
+            if ($($(content)[0]).css('display')!=='none'){
```

Comment:
To get the first element, we can use `content.first()`.

Reply:
ok iam using content.first() now